topics/timeseries.md

```diff
@@ -66,6 +84,10 @@ these columns depends on the `event_class` in question and is documented in deta
 
 ### Events with class `addToList`
 
+This event is emitted when a user has added a bookmark to a list. Only in classic
```

Comment:
This does also get submitted when a resource is added to a list straight from the user's my bookmarks page